Development server crashes on start due to SCSS parsing error

After cloning this repo and running `npm install`, I get the following error when trying to start up the development server:

```
» npm run dev

> isomorphic500@2.0.0 dev /Users/al3x/src/third_party/javascript/isomorphic500
> NODE_ENV=development DEBUG=isomorphic500 node index

  isomorphic500 Intl is not supported, so the polyfill has been loaded. +0ms
/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/babel/helpers/parse.js:70
    throw err;
          ^
SyntaxError: /Users/al3x/src/third_party/javascript/isomorphic500/src/style/LocaleSwitcher.scss: Unexpected token (1:0)
> 1 | @import 'constants/colors';
    | ^
  2 | 
  3 | .LocaleSwitcher-link {
  4 |   display: inline-block;
    at Parser.pp.raise (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/location.js:68:13)
    at Parser.pp.unexpected (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/parseutil.js:87:8)
    at Parser.pp.parseDecorator (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:146:10)
    at Parser.pp.parseDecorators (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:132:31)
    at Parser.pp.parseStatement (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:46:10)
    at Parser.parseStatement (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/plugins/flow.js:631:22)
    at Parser.pp.parseTopLevel (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:22:21)
    at Object.parse (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/index.js:109:12)
    at module.exports (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/babel/helpers/parse.js:39:21)
    at File.<anonymous> (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/babel/transformation/file/index.js:518:12)

npm ERR! Darwin 14.3.0
npm ERR! argv "/Users/al3x/.nvm/versions/io.js/v1.7.1/bin/iojs" "/Users/al3x/.nvm/versions/io.js/v1.7.1/bin/npm" "run" "dev"
npm ERR! node v1.7.1
npm ERR! npm  v2.7.6
npm ERR! code ELIFECYCLE
npm ERR! isomorphic500@2.0.0 dev: `NODE_ENV=development DEBUG=isomorphic500 node index`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the isomorphic500@2.0.0 dev script 'NODE_ENV=development DEBUG=isomorphic500 node index'.
npm ERR! This is most likely a problem with the isomorphic500 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=development DEBUG=isomorphic500 node index
npm ERR! You can get their info via:
npm ERR!     npm owner ls isomorphic500
npm ERR! There is likely additional logging output above.
```

FYI, that previous run was with iojs v1.7.1. Same issue happens with the latest version of Node:

```
» npm run dev

> isomorphic500@2.0.0 dev /Users/al3x/src/third_party/javascript/isomorphic500
> NODE_ENV=development DEBUG=isomorphic500 node index

  isomorphic500 Intl has been found, but locale data has been polyfilled. +0ms
/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/babel/helpers/parse.js:70
    throw err;
          ^
SyntaxError: /Users/al3x/src/third_party/javascript/isomorphic500/src/style/LocaleSwitcher.scss: Unexpected token (1:0)
> 1 | @import 'constants/colors';
    | ^
  2 | 
  3 | .LocaleSwitcher-link {
  4 |   display: inline-block;
    at Parser.pp.raise (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/location.js:68:13)
    at Parser.pp.unexpected (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/parseutil.js:87:8)
    at Parser.pp.parseDecorator (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:146:10)
    at Parser.pp.parseDecorators (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:132:31)
    at Parser.pp.parseStatement (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:46:10)
    at Parser.parseStatement (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/plugins/flow.js:631:22)
    at Parser.pp.parseTopLevel (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/statement.js:22:21)
    at Object.parse (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/acorn/src/index.js:109:12)
    at module.exports (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/babel/helpers/parse.js:39:21)
    at File.<anonymous> (/Users/al3x/src/third_party/javascript/isomorphic500/node_modules/babel-core/lib/babel/transformation/file/index.js:518:12)

npm ERR! Darwin 14.3.0
npm ERR! argv "/Users/al3x/.nvm/versions/node/v0.12.2/bin/node" "/Users/al3x/.nvm/versions/node/v0.12.2/bin/npm" "run" "dev"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4
npm ERR! code ELIFECYCLE
npm ERR! isomorphic500@2.0.0 dev: `NODE_ENV=development DEBUG=isomorphic500 node index`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the isomorphic500@2.0.0 dev script 'NODE_ENV=development DEBUG=isomorphic500 node index'.
npm ERR! This is most likely a problem with the isomorphic500 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=development DEBUG=isomorphic500 node index
npm ERR! You can get their info via:
npm ERR!     npm owner ls isomorphic500
npm ERR! There is likely additional logging output above.
```

--- a/index.js
+++ b/index.js
@@ -1,3 +1,4 @@
+delete process.env.BROWSER;
 
 // Register babel to have ES6 support on the server
 require("babel/register");

